pynipap/pynipap.py

```diff
@@ -147,7 +147,12 @@
     Classes
     -------
 """
-import xmlrpclib
+try:
```

Comment:
Explicitly check Python version (like performed in setup.py) instead of just blindly trying py3 import and failing over to py2 import.